src/copygb/copygb.F90

```diff
@@ -1190,7 +1190,7 @@ END SUBROUTINE INTGRIB
 SUBROUTINE INTGRIB1(K1F,KGDS1F,K2F,KGDS2F,MRL,MRO, &
      IV,IP,IPOPT,KGDS1,K1,IB1,L1,F1,G1,KGDS2,K2, &
      IB2,L2,F2,G2,IRET)
-  USE IP_MOD
+!  USE IP_MOD
```

Comment:
What's going on here?

Reply:
I've been doing a lot of fiddling in investigating CI issues, so this is related to restesting with the older ip version. I'm currently trying to figure why the intel version is segfaulting.

Reply:
OK, sorry I didn't notice that this was a draft PR.